CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 - #192 the async_nats crate has been merged into the
   main nats crate, under the `asynk` module name.
+- #192 the `jetstream::IntervalTree` type has been made
```

Comment:
I think we are leaning towards calling it OrderedConsumer.

Reply:
Should be done the Go client version this week, got delayed on server stuff.

Reply:
:+1: